src/main/java/com/example/rebrain/exception/CardNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.rebrain.exception;
+
+public class CardNotFoundException extends Exception {
```

Comment:
Let's have one common type for such exceptions

For instance, `ObjectNotFound` and etc.